src/item.cpp

```diff
@@ -8364,7 +8364,7 @@ bool item::use_amount( const itype_id &it, int &quantity, std::list<item> &used,
     // Remember quantity so that we can unseal self
     int old_quantity = quantity;
     std::vector<item *> removed_items;
-    for( item *contained : contents.all_items_top() ) {
+    for( item *contained : contents.all_standard_items_ptr() ) {
```

Comment:
it is really not appropriate to iterate through magazines and magazine_wells for items to be used; it really should only be CONTAINER type pockets; that being said, it can simply be a call to contents.all_items_ptr( item_pocket::pocket_type::CONTAINER )